data/service/api/v1/devicetokens.go

```diff
@@ -0,0 +1,45 @@
+package v1
```

Comment:
These tokens aren't used to authenticate to tidepool at all. I think it's better to keep the auth package as clean and distraction-free as possible, since that's important code.

All the code that uses these tokens are likely to live in the data package/service. So unless you're really certain, I think I like it where it is.

Reply:
I can see your argument against `auth` so I could see another microservice. However, this is not at all related to diabetes device data, so I feel pretty strongly it doesn't belong there. There *used* to be a `notification` microservice in `platform` where this would belong, but it was deleted in 2020. I have no idea why, though. @toddkazakov might know.

Reply:
@darinkrauss the notification service was deleted because it didn't expose any endpoints or provide any functionality. It was only slowing down build and deployment times. 

I also agree that this code doesn't belong to the data service. Those tokens are used for authorizing clients to push notifications to phones of Tidepool users, so placing the code in `auth` makes sense to me.

Reply:
OK, sounds good. I'll move it to auth.